developer-experience/recipes/onboarding-guide-template.md

```diff
@@ -0,0 +1,35 @@
+# Onboarding Guide Template
+
+When developing an onboarding document for a team, it should contain details of engagement scope, team processes, codebase, coding standards, team agreements, software requirements and setup details.
+
+## Overview and Goals
+
+* List a few sentences explaining the high-level summary and the scope of the engagement.
+* Consider adding any additional background and context as needed.
+* Include the goals of this engagement and what the team is trying to achieve.
+
+## Contacts
+
+* List a few of the main contacts from the engagement both from the customer and Microsoft team.
```

Comment:
Lets rewrite this within the context of one engineering team. We want this to be usable by any org even if they are not working on a CSE engagement. So maybe something like
```text
List a few of the main contacts for the team and project overall. [Such as ...]
```

I also added a `Such as...` placehoder at the end there. It would be helpful to have some guideance on roles that may be important here. I would think back on your own experience and who do you find yourself needing to interact with day-to-day to be effective. Tech lead and Product Owner are probably two obvious examples but, you may have some thoughts on others.